docs/current_docs/features/snippets/services-2/go/main.go

```diff
@@ -3,8 +3,6 @@ package main
 import (
 	"context"
 	"dagger/my-module/internal/dagger"
-
-	"dagger.io/dagger/dag"
```

Comment:
Oh woops - nice find. Actually, none of our code snippets should contain this, we should remove it everywhere.

Reply:
Pushed a commit with a few more of these I found.